docs/operations/installation.md

```diff
@@ -27,6 +27,59 @@ KubeVirt is currently supported on the following container runtimes:
 Other container runtimes, which do not use virtualization features,
 should work too. However, they are not tested.
 
+### Integration with AppArmor and SELinux
```

Comment:
nit: I don't see anything about SELinux in this section
```suggestion
### Integration with AppArmor
```

Reply:
Initially, I wanted to have a section about MACs in general. But then realized that there is basically nothing else to say about SELinux apart that KubeVirt works fine with it. So this is currently just emphasized in the first sentence:
> "In most of the scenarios, KubeVirt can run normally on systems with AppArmor or **SELinux**."

Still not sure if it makes sense to mention SELinux at all. I will probably drop it then, unless there are other thoughts on that...